ReleaseChecklist.md

```diff
@@ -32,6 +32,10 @@
 ### Homebrew and MacOS
  - [ ] Update the version and the hash (``sha256sum solidity_x.x.x.tar.gz``) in https://github.com/ethereum/homebrew-ethereum/blob/master/solidity.rb
  - [ ] Take the binary from the ``b_osx`` run of the released commit in circle-ci and add it to the release page as ``solc-macos``.
+ - [ ] bump new homebrew formula 
```

Comment:
is it possible to do this without the `brew` command?

Reply:
we could write a script

Reply:
What does it do?

Reply:
maybe just checking out the last formula, changing it, git add, git commit, git push.. but i didnt looked into what `brew` is actually doing, maybe there is more magic involved

Reply:
@ekpyron also found https://github.com/marketplace/actions/homebrew-bump-formula, maybe that would be easier to use

Reply:
Yeah... The action apparently was written by https://github.com/dawidd6 who appears to be homebrew maintainer, so that doesn't look bad - and we could just have it be triggered by the release announcement being published - so that could be rather convenient... but yeah, we might also get away with manually creating PRs bumping version and hashes (the brew command does some verification, maybe even a test build, but ultimately just created this PR: https://github.com/Homebrew/homebrew-core/pull/60549 - but the conversations @axic linked earlier sounded like the brew command is the preferred way they want people to use to create these PRs...).

Reply:
But yeah, I'd say let's first just watch that PR and if and how fast it gets merged and then decide how to proceed and do things in the future.

Reply:
The PR was merged, see https://github.com/Homebrew/homebrew-core/commit/cfc449fdde19b61953777c10677c8b2f4d002d81. 

I think it would be nice to do this somehow automated, but if not: there seem to be linuxbrew docker images available https://github.com/Linuxbrew/docker. maybe we could do such PR's manually without needing macos. However, I tried `docker run -it linuxbrew/alpine` and it seem to be a bit buggy - it was just not simply working as expected ;) there where some strange errors, i didn't investigate.. but maybe after some research we could also use this

Reply:
For the record: it only took them around two hours to merge it I think. Also apparently they do automatically build the core formulas to "bottles" (i.e. binaries), which is nice as well - the osx builds should be deterministic, shouldn't they? We could probably verify that the bottle hashes match our builds (in case we deem it necessary).

@aarlt I think the docker image is for "linux brew", i.e. the version of homebrew to be used as linux package manager - I'm not sure linux brew can be used for updating osx packages...

Reply:
@ekpyron yes, you're right.. I thought that these recipes are platform independent.. but at least for the bottled part it should not work